src/migrations/20230116162612-add-sats-amounts-to-receipts.ts

```diff
@@ -0,0 +1,90 @@
+const txTypes = ["invoice", "onchain_receipt"]
+
+module.exports = {
+  async up(db) {
+    const collection = db.collection("medici_transactions")
+
+    try {
+      const allTxns = await collection.aggregate([
+        {
+          $match: {
+            type: { $in: txTypes },
+            satsAmount: { $exists: false },
+            currency: "BTC",
+          },
+        },
+        {
+          $group: {
+            _id: "$hash",
```

Comment:
same hash could apply to different receipt transactions, this will add wrong values to some transactions, maybe the best option is to group by journal i

Reply:
Fixed https://github.com/GaloyMoney/galoy/commit/0b22560c2f07bc447444b5bd561700833aea0fd6